### PR TITLE
add french locale

### PR DESCRIPTION
**Pull request by fat115**

### CODE REVIEW

File: youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Stopping downloads"
+msgstr "Arrêt des téléchargements"
+
+#: youtube_dl_gui/mainframe.py:122
+msgid "Downloads stopped"
+msgstr "Téléchargements arrêtés"
+
+#: youtube_dl_gui/mainframe.py:123
+msgid "You need to provide at least one URL"
+msgstr "Vous devez fournir au moins un lien"
```
Comment (nodiscc) [inline youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po]: `Vous devez fournir au moins une URL` ?

File: youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "Mise à jour de youtube-dl. Merci de patienter"
+
+#: youtube_dl_gui/mainframe.py:131
+msgid "Youtube-dl download failed [{0}]"
+msgstr "Echec de la mise à jour de Youtube-dl [{0}]"
+
+#: youtube_dl_gui/mainframe.py:132
+msgid "Successfully downloaded youtube-dl"
+msgstr "Mise à jour réussie de youtube-dl."
```
Comment (nodiscc) [inline youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po]: `Mise à jour de youtube-dl réussie` ?

File: youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po [hunk 3]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Extension"
+msgstr "Extension"
+
+#: youtube_dl_gui/mainframe.py:142
+msgid "Size"
+msgstr "Taille"
+
+#: youtube_dl_gui/mainframe.py:143
+msgid "Percent"
+msgstr "%"
```
Comment (nodiscc) [inline youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po]: `Pourcents` ?

File: youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po [hunk 4]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Speed"
+msgstr "Vitesse"
+
+#: youtube_dl_gui/mainframe.py:146
+msgid "Status"
+msgstr "État"
+
+#: youtube_dl_gui/mainframe.py:235
+msgid "Get URL"
+msgstr "Récupérer le lien"
```
Comment (nodiscc) [inline youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po]: `Récupérer l'URL` ?

File: youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po [hunk 5]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Item is active, cannot remove"
+msgstr "Élément actif, ne peut être enlevé"
+
+#: youtube_dl_gui/mainframe.py:579
+msgid "Item is not completed"
+msgstr "Élément incomplet"
+
+#: youtube_dl_gui/mainframe.py:668
+msgid "Update in progress. Please wait for the update to complete"
+msgstr "Mise à jour en cours. Veuillez attendre que la mise à jour soit complète"
```
Comment (nodiscc) [inline youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po]: `Veuillez attendre que la mise à jour soit terminée` ?

File: youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po [hunk 6]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Youtube-dl command line options (e.g. --help)"
+msgstr "Options de ligne de commande Youtube-dl (ex : --help)"
+
+#: youtube_dl_gui/optionsframe.py:866
+msgid "Extra options"
+msgstr "Options supplémentaires"
+
+#: youtube_dl_gui/optionsframe.py:868
+msgid "Debug youtube-dl"
+msgstr ""
```
Comment (nodiscc) [inline youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po]: `Débugger youtube-dl` ?

File: youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po [hunk 7]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Ignore errors"
+msgstr "Ignorer les erreurs"
+
+#: youtube_dl_gui/optionsframe.py:870
+msgid "Ignore youtube-dl config"
+msgstr "Ignorer config youtube-dl"
+
+#: youtube_dl_gui/optionsframe.py:871
+msgid "No mtime"
+msgstr ""
```
Comment (nodiscc) [inline youtube_dl_gui/locale/fr_FR/LC_MESSAGES/youtube_dl_gui.po]: `Pas de mtime` ?

**Comment by fat115** [comment]
Translation updated according to your proposals.
.mo file added ... using git add --force => cf. .gitignore

**Comment by MrS0m30n3** [comment]
Excellent work as @nodiscc said but you should remove the *.mo* file.
Translators should not commit those since we build them during the package build phase.

**Comment by fat115** [comment]
.mo file removed as requested (it was added when @nodiscc requested it ;) )

**Comment by MrS0m30n3** [comment]
I know, the 7th step it the guide says to build the binary translation files (.MO), but that's only for the translator to test his work. I guess i should clarify this in the guide.

Anyway, everything is fine now. I will merge this as soon as i finish with the other translations.

**Comment by nodiscc** [comment]
> .mo file was added when @nodiscc requested it
> the 7th step in the guide says to build the binary translation files (.MO), but that's only for the translator to test his work. I guess i should clarify this in the guide.

Sorry! I found this confusing as well.

Fine translation @fat115  :+1:

**Pull request merged**